proto/iotics/api/feed.proto

```diff
@@ -44,9 +44,9 @@ service FeedAPI {
 }
 
 // A feed representation.
-message Feed {
-  // Feed identifier (unique within the scope of a twin identifier's feed set)
-  FeedID id = 1;
+message FeedID {
+  // Feed identifier string representation (simple string) (unique within the scope of a twin identifier's feed set)
+  string id = 1;
   // Twin unique identifier (twin to which the feed belongs)
   TwinID twinId = 2;
```

Comment:
Here and in Input we could have the flatten twinId and hostId instead of the `TwinID` object